META.json

```diff
@@ -171,10 +170,14 @@
       "David Morton <davidlmorton@gmail.com>",
       "David Morton <davidlmorton@users.noreply.github.com>",
       "Sebastian Heil <(none)>",
-      "APipe Tester <apipe-tester@genome.wustl.edu>",
       "nnutter <nnutter@genome.wustl.edu>",
       "Nathaniel Nutter <iam@nnutter.com>",
+      "Eddie Belter <ebelter@genome.wustl.edu>",
+      "Eddie Belter <ebelter@gmail.com>",
+      "Jim Eldred <jeldred@genome.wustl.edu>",
+      "Yanick Champoux <yanick@babyl.dyndns.org>",
       "Thomas B. Mooney <tmooney@genome.wustl.edu>",
+      "Thomas B. Mooney <mooney@wustl.edu>",
```

Comment:
Yeah, they're put in automatically by the ```minil``` release tool by looking at email addresses in the commit history.  We could take them out, but they'd show up again the next time it gets released if we stay with ```minil```.